scss/utils/vars/_scale.scss

```diff
@@ -88,6 +88,8 @@ $space-quarter   : $space/4;
 * can be thought of as "media". The following variables
 * set media sizes based on our modular scale:
 */
+/// @type Value(number)
+$mediaScale: 1;
```

Comment:
I think we might actually lose some clarity about how this variable is used by defining it here.

We should just hard code `1` where `--mediaScale` is defined.

Right now our definitions look something like this:
`$sassVar -> --customProp: $sassVar;`

Moving forward, we'll probably want to do something more like this:
`--customProp: actualValue;`
`--customPropReferencingOtherProp:  var(--customProp);`